### DISCUSSION TITLE
2024-01-24 Weekly Hosting Meeting

### DISCUSSION THREAD

**Discussion by javiercasares** [WordPress/hosting-handbook#214] [Weekly Meetings]
Hosting Team meeting agenda 2024-01-10
---

EXCERPT:

PUBLIC AGENDA URL:

⇾ 

PUBLIC SUMMARY URL (if needed):

⇾ 

# Agenda

WordPress Community
- WordPress Core news
- WordPress Community news

Hosting Team Time
- WordPress Hosting Compatibility
- Team Goals for 2024
- WordPress Hosting Team event
- More Hosting Team topics

Open Floor
- Anything not in the agenda

# Meeting

/here [meeting type="hosting"] Who's here to chat about hosting?

👋 Hello! This is the meeting for the WordPress.org Hosting Team, who works with hosts and developers to help improve the WordPress user experience. This is a great time to ask any questions you may have, or visit [Make/Hosting](https://make.wordpress.org/hosting/) for more info.

🥇 If you participate in the [Five for the Future](https://wordpress.org/five-for-the-future/) program, drop your WordPress.org username in a thread here 👉

🤗 If you're new here, welcome! You are welcome to introduce yourself if you like. 👉

## Agenda

🟢 Here is the agenda for this meeting

[...] Use the agenda above [...]

Would anyone like to add to or change any items on the agenda?

OK, let's get started with the WordPress Community news.

### 🟢 WordPress Core news

[WordPress 6.4.3 Release](https://wp.me/p2AvED-sLg) 
There is an upcoming maintenance release for WordPress. The RC is scheduled to be released tomorrow, Jan 25th, 2024 and the final version is planned for Jan 30th, 2024. This is a maintenance release focused on fixing some bugs which are currently present in 6.4.2. 

[Roadmap to 6.5](https://make.wordpress.org/core/2023/12/07/roadmap-to-6-5/).
The core team published the roadmap to the next big release of WordPress. Be prepared for WordPress 6.5 which is planned to be released on March 26, 2024. 

[Raising the minimum version of MySQL required in WordPress 6.5](https://make.wordpress.org/core/2023/12/08/raising-the-minimum-version-of-mysql-required-in-wordpress-6-5/).
With WordPress 6.5 there will also be a bump in the minimal version of MySQL that will be required. The version jump will be from `5.0` to `5.5.5`. Please note that this is the minimum version required, the recommendation is to frequently update your stack. MySQL 5.5 was chosen as this will cover 99.6% of al WordPress installations according to the stats (https://wordpress.org/about/stats/). 

### 🟢 WordPress Community news

General information

[Big Picture for 2024](https://make.wordpress.org/project/2024/01/19/big-picture-goals-2024/)
Josepha shared the big goals for the community for 2024 with a focus on three core topics:
**CMS**: [Test, iterate, and ship](https://make.wordpress.org/core/) Phase 3 of the Gutenberg project
**Community**: Continue to support the community through [learning](https://learn.wordpress.org/), [events](https://events.wordpress.org/), and [mentorship](https://make.wordpress.org/community/tag/mentorship-program/) of current and future contributors
**Ecosystem**: Address the difficulty in moving platforms through the [Data Liberation project](https://wordpress.org/news/2024/01/data-liberation-in-2024/) as well as streamline existing review processes across repositories

[Data Liberation](https://wordpress.org/data-liberation/) project. @matt explained more about the Data Liberation project that got announced during the State of the Word 2023 in Madrid. We as a hosting team can support there especially with insights of our customer base and how the domain movement is from one system to another. If you have some insights you can share, please go ahead. 

[Theme Updates](https://wp.me/p2AvED-sLF). The previous yearly themes received an update to be better supported by PHP 8. More information is included in the post. 

[Developer Hours February](https://www.meetup.com/learn-wordpress-online-workshops/events/298634428/) are back. In February the core team wants to focus on JavaScript for modern WordPress development. The Developer Happy Hour is a one-hour live session around a specific topic for WordPress Developers offered by the WordPress community.  It takes place on Feb 13, 2024 from 4 to 5 PM UTC.

2023 summary

- [WordPress End of Year Celebrations!](https://make.wordpress.org/project/2023/12/22/wordpress-end-of-year-celebrations/).
- [WordPress performance impact on Core Web Vitals in 2023](https://make.wordpress.org/core/2023/12/19/wordpress-performance-impact-on-core-web-vitals-in-2023/)

Events information

- [WordCamp Asia Contributor Day](https://asia.wordcamp.org/2024/contributor-day/).
- [WordCamp Europe Tickets](https://europe.wordcamp.org/2024/tickets/) are online.
- [WordCamp US 2024](https://us.wordcamp.org/2024/) September 17-20, 2024 (Portland) is announced
Is anyone joining a local WordCamp with a contributor day and wants to focus on Hosting? Let us know how we can support you best! 

Additional News
- [Global Sponsors for 2024](https://make.wordpress.org/community/2024/01/09/2024-global-sponsors-announcement/)
- [Call for Mentees & Mentors - Cohort 2](https://make.wordpress.org/community/2024/01/12/call-for-mentees-mentors-cohort-2/) 

Ok, let's move on to Hosting Team time.

### 🟢 WordPress Hosting Compatibility

Meeting reminder, only for your information:

- [WordPress 6.4 PHP Compatibility](https://make.wordpress.org/hosting/2023/11/16/wordpress-6-4-php-compatibility/)
- [WordPress 6.3 PHP Compatibility](https://make.wordpress.org/hosting/2023/10/11/wordpress-6-3-php-compatibility/)
- [WordPress Hosting Server Environment](https://make.wordpress.org/hosting/handbook/server-environment/)
- [WordPress Compatibility](https://make.wordpress.org/hosting/handbook/compatibility/)

### 🟢 Team Goals for 2024

We want your input on the goals we're planning for this team for 2024. There will soon be a post on our p2 about the goals, and you can already start brainstorming your ideas on this channel. Our main ideas are to focus on 4 areas:

1. Testing Project with the Hosting Tests

- Continuation Plan: Enhancing our [PHPUnit test runner](https://github.com/WordPress/phpunit-test-runner) and [PHPUnit test reporter](https://github.com/WordPress/phpunit-test-reporter) by adding new capabilities and moving it to the latest tech stack.
- Community Involvement: Inviting more hosting companies to participate and share their insights to enrich this project.
- Improve communication: Contact all the testers to see if the team can help improve the information and feedback.

2. Documentation Projects

- Advanced Admin Documentation: Updating and expanding our resources to cover the latest trends and best practices in WordPress administration.
- Hosting Handbook: Continually refining our handbook to make it an even more comprehensive guide for both new and veteran WordPress hosts.
- Migrate content: There is some documentation in the Hosting Handbook that should be in the Advanced Administration or, maybe, other handbooks.

3. The Hosting Event 2024

- Event Planning: Organizing a hosting-centric WordPress event in June 2024 (before WordCamp Europe 2024).

4. CloudFest Hackathon 2024

- The [WordPress Tools for Hosting Providers](https://hackathon.cloudfest.com/project/enhanced-wordpress-unit-testing-suite/) project has been selected to participate on March 16-18, 2024 in the CloudFest Hackathon.

### 🟢 WordPress Hosting Team event

We've met two weeks ago to discuss the ideas around the WordPress Hosting Team event, which is planned to take place the day before the Contributor Day of #wceu in 2024. You can expect more information soon.

Floating ideas:
- Event about hosting, security, performance, and sustainability.
- Event date: 2023-06-12 (so, Wednesday before Contributor Day / WCEU)
- Event time around 0800-1400 (this will leave the afternoon free, so there is time to visit the city or go to WCEU sponsor events)
- One track, but two kinds of talks: “Community” and “Professional”. “Community” from the community to the attendees, so we can show WordPress projects and how Host can benefit and contribute. “Professional” so professionals can give us some great talks about what they are doing around hosting, security, performance, and sustainability…

When we have some OK from different parts involved, and we can start, officially, we will announce everything and start detailing more. We already have a location in Torino that would be available. 

#### 🟩 WP 6.5 and PHP compatibility

WordPress 6.5 is expected in early 2024 (2024-03-26), and we can already prepare us from the hosting site. Please make sure to always test the latest builds and help the community by providing feedback. 

Let's make sure to have a compatibility and recommendation post around WordPress 6.5 on the release, with a focus on PHP 8.2 and 8.3 until then.

#### 🟩 Team meetings

We don't have so much news to have 2 weekly meetings. The focus is to work more asynchronously.

But, there will be an exception. Each new major version, we will have a meeting the next day around 0900 UTC, probably with the _WordPress N.N PHP Compatibility_ post related.

Dates are:

- 2024-02-07 1800 UTC
- 2024-02-21 0900 UTC
- 2024-03-06 1800 UTC
- 2024-03-20 0900 UTC
- 2024-03-27 0900 UTC (tentative, WordPress 6.5 release)

All the meetings will be announced at the [Meetings](https://make.wordpress.org/meetings/#hosting) site. You can subscribe to it ([iCal Hosting](https://make.wordpress.org/meetings-hosting.ics).

#### 🟩 Office hours

Same as we are working on reducing the weekly meetings, the focus for this 2024 is to increase the work on Documentation and Tools, and support for Hosting providers.

We will propose a calendar for those meetings.

Topics:
- Documentation ([Hosting Team Handbook](https://make.wordpress.org/hosting/handbook/))
- Documentation ([Advanced Admin Handbook](https://developer.wordpress.org/advanced-administration/))
- Tools ([Test Runner](https://github.com/WordPress/phpunit-test-runner) aka test-runner)
- Tools ([API and Results Page](https://github.com/wordpress/phpunit-test-reporter) aka test-reporter)

The idea is to have at least a weekly meeting about one of these topics.

#### 🟩 Tickets

Hosting Team

- https://meta.trac.wordpress.org/ticket/7371

General hosting-related tickets

- https://meta.trac.wordpress.org/ticket/7236
- https://core.trac.wordpress.org/ticket/56390
- https://core.trac.wordpress.org/ticket/58816
- https://core.trac.wordpress.org/ticket/54116
- https://core.trac.wordpress.org/ticket/54338
- https://core.trac.wordpress.org/ticket/50188

Test tools

- https://github.com/WordPress/phpunit-test-reporter/pulls
- https://github.com/WordPress/phpunit-test-runner/pulls

## 🟩 Open Floor

Ok, if nothing else, let's move on to the Open Floor. You are welcome to bring any topic up for discussion at this time.

...

## Goodbye

Okay, let's close out this meeting.

You are welcome to continue the conversation here in the [Slack #hosting](https://wordpress.slack.com/archives/C3D6T7F8Q) channel, or on [Make/Hosting](https://make.wordpress.org/hosting/), at any time.

Here are the team's [upcoming meetings](https://make.wordpress.org/meetings/#hosting):
- 2024-02-07 1800 UTC
- 2024-02-21 0900 UTC
- 2024-03-06 1800 UTC
- 2024-03-20 0900 UTC
- 2024-03-27 0900 UTC (tentative, WordPress 6.5 release)

Thank you all for joining and for your contributions to WordPress!

Hope you have a wonderful rest of your week! [/meeting]

**Comment by javiercasares** [comment]
The document is a copy from the last meeting… needs updating.

**Reply by Crixu** [reply]
reworked  most of it

**Comment by javiercasares** [comment]
News:
- https://make.wordpress.org/community/2024/01/12/call-for-mentees-mentors-cohort-2/
- https://make.wordpress.org/community/2024/01/09/2024-global-sponsors-announcement/
- https://us.wordcamp.org/2024/
- https://make.wordpress.org/project/2024/01/19/big-picture-goals-2024/
- https://make.wordpress.org/core/2024/01/18/wordpress-6-5-release-squad-formation/
- https://make.wordpress.org/core/2024/01/16/6-4-3-an-upcoming-maintenance-release/

**Comment by javiercasares** [comment]
## PHPUnit Test Tools


The [WordPress Tools for Hosting Providers project](https://hackathon.cloudfest.com/project/enhanced-wordpress-unit-testing-suite/) has been selected to participate on March 16-18, 2024 in the CloudFest Hackathon.

For this reason, we have 5 goals for that date.

The first is to leave a stable, functional, and documented version of the _test-runner_ tool so that everyone works on that version. There is a [PR #199](https://github.com/WordPress/phpunit-test-runner/pull/199) that establishes those changes and that base. It has been thoroughly tested and should serve as a basis so that the errors we have previously encountered are not there.

The second is to do the same work that has been done on the _test-runner_ but on the _test-reporter_ so that we also have a more or less stable version to work on the plugin.

The third is to leave documentation of the entire operation of the tools and how to set them up in a development environment, as well as how they can be tested (each tool will need its own explanation, since one is best to have a very "hosting" environment, and for the plugin, to have a WordPress development environment of its own).

The fourth, which involves everyone, is to define a list of "what things we want the tools to have." In principle, there should be two lists, which must complement each other. One that executes the test-runner and allows executing "things", and the other of the test-reporter that collects all that and displays it on the screen, in addition to planning warnings.

The fifth and last is to say that those who want to come to hack a bit, give a read to [Come Hack with Us!](https://hackathon.cloudfest.com/attendee-info-registration/). We look forward to seeing you March 16-18, 2024, especially to have a great time!

### Test Tools Office hours

@JavierCasares is working on these goals, and he will be available, and scrubbing tickets, on this agenda:

- 2024-01-31 0900 UTC
- 2024-02-07 0900 UTC
- 2024-02-14 0900 UTC
- 2024-02-28 0900 UTC

**Comment by javiercasares** [comment]
## Contributor Day

- 2024-03-07: [WordCamp Asia 2024 Contributor Day](https://asia.wordcamp.org/2024/)
- 2024-06-13: [WordCamp Europe 2024 Contributor Day](https://europe.wordcamp.org/2024/)
- 2024-09-17/18: [WordCamp US 2024 Contributor Day](https://us.wordcamp.org/2024/)
